client/a8c-for-agencies/sections/marketplace/hosting-overview/hooks/use-pressable-ownership-type.ts

```diff
@@ -13,9 +13,8 @@ export default function usePressableOwnershipType() {
 			return 'none';
 		}
 
-		// If the agency has a regular Pressable plan (not brought through A4A marketplace), A4A id is null.
 		const hasRegularPressablePlan =
-			hasPressablePlan && activeAgency?.third_party?.pressable?.a4a_id === null;
+			activeAgency?.third_party?.pressable?.subscription?.status === 'active';
```

Comment:
To confirm, will this only be `status === 'active'` when it's a regular Pressable plan that wasn't bought through A4A? Like Pressable reports something else for A4A-initiated subscriptions?

Reply:
I thought so, but it seems not always the case. I think I need to check both. Also dropped a question here: p1733870548380059/1733439000.969209-slack-C07LKD3JU91